cfgov/teachers_digital_platform/jinja2/teachers_digital_platform/survey/results-6-8.html

```diff
@@ -236,13 +236,13 @@ <h4>Stop and think:</h4>
         <div class="o-well block">
             <h2>Three things you can do to get farther down the road on your money journey</h2>
 
-            <h3>① Explore your strengths</h3>
+            <h3 class="numbered-header"><img src="{{ static('apps/teachers-digital-platform/img/icon-one.svg') }}" alt="1."> Explore your strengths</h3>
```

Comment:
For icons elsewhere on the site we put the svg directly into the HTML. Would it make sense to do that here?